integreat_cms/cms/templates/_search_input.html

```diff
@@ -8,14 +8,17 @@
                name="query"
                autocomplete="off"
                id="table-search-input"
+               aria-haspopup="listbox"
+               aria-label="Search"
                placeholder="{% translate "Search" %}"
                class="default-value rounded-r-none"
                data-url="{% if region_slug and language_slug %}{% url 'search_content_ajax' region_slug language_slug %}{% elif region_slug %}{% url 'search_content_ajax' region_slug %}{% else %}{% url 'search_content_ajax' %}{% endif %}"
                data-object-type="{{ object_type }}"
                data-archived="{% if object_archived %}true{% else %}false{% endif %}" />
-        <div id="table-search-suggestions"
-             class="absolute hidden shadow rounded-b top-full bg-graz-200 w-full z-10 max-h-60 overflow-y-auto cursor-pointer">
-        </div>
+        <ul id="table-search-suggestions_list"
```

Comment:
Is there an intention to use an underscore before the list in "table-search-suggestions_list"? :)

Reply:
@JoeyStk Thank you for your review! :) I wanted to highlight that it is a list, but there is no extra intention...

Reply:
@lunars97 Thank you for explaining! No worries, that was not what I meant. I was wondering, why you were changing from dash "table-search-suggestions" to underscore "_list" :)

Reply:
@JoeyStk I think I will remove it better, because the id is self explanatory :)